server/src/test/java/org/elasticsearch/cluster/metadata/MetadataIndexTemplateServiceTests.java

```diff
@@ -727,7 +749,6 @@ public void testFindV2InvalidGlobalTemplate() {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/pull/57393")
```

Comment:
This seems to be related to composable templates, the mentioned PR was abandoned though. Is this really fixed? Or was this muted by accident initially?

Reply:
It was fixed in the past with Julie's mappings merging, the test was just never unmuted (it's fixed now though)